src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigurationFilesControl.xaml.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Input;
+
+namespace NuGet.PackageManagement.UI.Options
+{
+    /// <summary>
+    /// Interaction logic for ConfigurationFilesControl.xaml
+    /// </summary>
+    public partial class ConfigurationFilesControl : UserControl
+    {
+        private ConfigurationFilesViewModel _viewModel;
+
+        public ConfigurationFilesControl(ConfigurationFilesViewModel configurationFiles)
+        {
+            InitializeComponent();
+            _viewModel = configurationFiles;
+            DataContext = configurationFiles;
+        }
+
+        private void ListViewItem_MouseDoubleClick(object sender, MouseButtonEventArgs e)
+        {
+            _openButton.Command.Execute(null);
+        }
+
+        internal void InitializeOnActivated()
+        {
+            _viewModel.SetConfigPaths();
+        }
+
+        private void ExecuteOpenExternalLink(object sender, ExecutedRoutedEventArgs e)
+        {
+            if (e.OriginalSource is Hyperlink hyperlink && hyperlink.NavigateUri != null)
+            {
+                UIUtility.LaunchExternalLink(hyperlink.NavigateUri);
+
+                e.Handled = true;
+            }
+        }
```

Comment:
Based on my [comment above](https://github.com/NuGet/NuGet.Client/pull/5292/files#r1273996057), we should instead have a RequestNavigate handler for the Hyperlink that does the following:

```suggestion
        private void Hyperlink_OnRequestNavigate(object sender, RequestNavigateEventArgs e)
        {
            UIUtility.LaunchExternalBrowser(e.Uri.AbsoluteUri);
            e.Handled = true;
        }
```

And then the Hyperlink in the XAML should have the `RequestNavigate` attribute pointing to `Hyperlink_OnRequestNavigate` to hook up this event handler.